test/bench/bench.cpp

```diff
@@ -175,15 +174,27 @@ void load_benchmark(fs::path path)
     }
 }
 
-void load_benchmarks_from_dir(const char* path)
+void load_benchmarks_from_dir(const fs::path& path, const fs::path& subdir = {})
```

Comment:
Maybe it's `(const fs::path& absolute_path, const fs::path& relative_path = {})`

Reply:
It is actually not. The first one can be also an relative path. Refactored differently as suggested in other comment.